projects/packages/connection/tests/php/test_Manager_unit.php

```diff
@@ -403,10 +403,10 @@ public function test_get_signed_token() {
 		$access_token->secret = 'abcd.1234';
 
 		$signed_token = ( new Tokens() )->get_signed_token( $access_token );
-		$this->assertTrue( strpos( $signed_token, 'token' ) !== false );
-		$this->assertTrue( strpos( $signed_token, 'timestamp' ) !== false );
-		$this->assertTrue( strpos( $signed_token, 'nonce' ) !== false );
-		$this->assertTrue( strpos( $signed_token, 'signature' ) !== false );
+		$this->assertTrue( str_contains( $signed_token, 'token' ) );
+		$this->assertTrue( str_contains( $signed_token, 'timestamp' ) );
+		$this->assertTrue( str_contains( $signed_token, 'nonce' ) );
+		$this->assertTrue( str_contains( $signed_token, 'signature' ) );
```

Comment:
Probably these should just use `$this->assertStringContainsString()`.

Same elsewhere where `assertTrue( str_contains() )` or `assertFalse( str_contains() )` are being used.

Reply:
Done in 336c6aef88e274c3d2c7cec0b297a8d6512a8bf0